tests/v2/test_wakunode.nim

```diff
@@ -109,3 +109,37 @@ procSuite "WakuNode":
     check:
       (await completionFut.withTimeout(5.seconds)) == true
     await allFutures([node1.stop(), node2.stop()])
+
+  asyncTest "Store protocol returns expected message":
+    let
+      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node1 = WakuNode.init(nodeKey1, ValidIpAddress.init("0.0.0.0"),
+        Port(60000))
+      nodeKey2 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node2 = WakuNode.init(nodeKey2, ValidIpAddress.init("0.0.0.0"),
+        Port(60002))
+      contentTopic = "foobar"
+      message = WakuMessage(payload: "hello world".toBytes(), contentTopic: contentTopic)
+      uuid = "123"
+
+    var completionFut = newFuture[bool]()
+
+    await allFutures([node1.start(), node2.start()])
+
+    await node2.subscriptions.notify("", message)
```

Comment:
What does empty string as argument mean in this context? Seems weird to me. Is it supposed to be a random string like "foobar"? Or is this some catchall kind of behavior (should be nil/all type aliased then, or a wrapper function)?